pipelines/visitQualityCore.yaml

```diff
@@ -46,3 +46,87 @@ tasks:
     config:
       atools.astromColorDiffMetrics: TargetRefCatDeltaColorMetrics
       python: from lsst.analysis.tools.atools import *
+  calexpSummary:
+    class: lsst.analysis.tools.tasks.CalexpSummaryAnalysisTask
+    config:
+      atools.calexpSummaryMetrics: CalexpSummaryMetrics
```

Comment:
Once DM-43077 merges, there won't be any more references to a `calexp`. We should rethink these names, perhaps use `pviSummary` and `pviSummaryMetrics`. Maybe get some naming suggestions from #dm-naming-things, since I don't love the `pvi` suggestions either.

Reply:
I've left the names as the are for now, but have created a ticket (DM-46932) to rename/delete/whatever as part of DM-43077.